recipes/rnaz/meta.yaml

```diff
@@ -18,7 +18,6 @@ requirements:
     - {{ compiler('cxx') }}
   host:
     - libstdcxx-ng
-    - libsvm =3.21
```

Comment:
Can you remove this strict perl pinning please?

Reply:
Removed the pinning and it still builds